### PR TITLE
## In this pull request the following was done 

### PR DESCRIPTION
**Pull request by Tchilo**
- Created an original website design where images are from the Unsplashed API
- Utilized async and await along with ES6 JavaScript modules
- Solved git conflicts and performed code reviews for the team
- Individually built interfaces (home page and comment pop-up sections)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 MIT License
+
+This project makes use of the MIT license.
```
Comment (DeliceLydia) [inline README.md]: - [x] Please record a video explaining features you implemented in this project and the maximum length of this project is 5min and add it in the README
- [x] All members should appear in the video and participate
- [x] The video contains a highlight about a piece of the code you find interesting or are proud of making.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Tchilo) [reply README.md]: Video link has been added 😃

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    });
+  }
+};
+
+const openModal = () => modal.classList.add('show-modal');
+
+const checModal = async (e) => {
+  const item = e.target;
+  const displayData = await getImages();
+  console.log(displayData);
```
Comment (Davidosky007) [inline src/index.js]: * Kindly remove all `console.log` code from your codebase as it is not good practice sensitive information might get logged in the console

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  });
+};
+
+const getImages = async () => {
+  const response = await fetch(url);
+  const data = await response.json();
+  return data.results;
+};
+
+const displayDom = async () => {
```
Comment (Davidosky007) [inline src/index.js:77]: * Kindly ensure Counters are present in all the interfaces that show the number of items (home) as stated in the project requirement [Here](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=Counters%20We%20have,of%203%20students.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Tchilo @VICTOR-CHUKWU 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
* As mentioned by the previous code reviewer your video presentation should contain a highlight about a piece of code you found interesting or are proud of making 👍 
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by de-scholar — APPROVED** [review]
Hi @Tchilo and @VICTOR-CHUKWU , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**